bundles/org.openhab.persistence.jdbc/src/main/java/org/openhab/persistence/jdbc/internal/db/JdbcSqliteDAO.java

```diff
@@ -130,9 +132,11 @@ public void doStoreItemValue(Item item, State itemState, ItemVO vo, ZonedDateTim
         ItemVO storedVO = storeItemValueProvider(item, itemState, vo);
         String sql = StringUtilsExt.replaceArrayMerge(sqlInsertItemValue,
                 new String[] { "#tableName#", "#dbType#", "#tablePrimaryValue#" },
-                new String[] { formattedIdentifier(storedVO.getTableName()), storedVO.getDbType(), "?" });
+                new String[] { formattedIdentifier(storedVO.getTableName()), storedVO.getDbType(),
+                        sqlTypes.get("tablePrimaryValueParsed") });
```

Comment:
I'm not sure that will work, since you added `tablePrimaryValueFormated` rather than `tablePrimaryValueParsed`?

Reply:
Thank you for reviewing and sorry for the dummy mistake (decided that the name ending with `Parsed` isn't good and change it while committing, but missed the other space and forgot to test it again ...).